quinn/src/runtime.rs

```diff
@@ -84,6 +84,11 @@ pub fn default_runtime() -> Option<Arc<dyn Runtime>> {
         return Some(Arc::new(AsyncStdRuntime));
     }
 
+    #[cfg(feature = "runtime-smol")]
+    {
+        return Some(Arc::new(SmolRuntime));
+    }
+
     #[cfg(not(feature = "runtime-async-std"))]
```

Comment:
Does this need to be revised to `not(any(feature = "runtime-async-std", feature = "runtime-smol"))` to avoid issues when only `runtime-smol` is enabled?